x-pack/plugins/uptime/public/components/overview/monitor_list/use_monitor_histogram.test.tsx

```diff
@@ -17,7 +17,7 @@ describe('useMonitorHistogram', () => {
   jest
     .spyOn(reduxHooks, 'useSelector')
     .mockReturnValue({ settings: { heartbeatIndices: dynamicIndexPattern } });
-  jest.spyOn(searchHooks, 'useEsSearch').mockImplementation(useEsSearch);
+  jest.spyOn(searchHooks as any, 'useEsSearch').mockImplementation(useEsSearch);
```

Comment:
Locally I was getting an error like:
`TS2590: Expression produces a union type that is too complex to represent`